package.json

```diff
@@ -85,7 +78,8 @@
     "react-hero-icon": "^0.0.7",
     "react-scripts": "3.4.4",
     "storybook-addon-designs": "^5.4.3",
-    "storybook-dark-mode": "^1.0.4"
+    "storybook-dark-mode": "^1.0.4",
+    "tailwindcss-classnames": "^2.0.1"
```

Comment:
I went with just `classnames` as the benefits of using it requires you to configure it with our custom tailwind.config.js file. You won't get any of our custom stuff without doing that.

Reply:
Yeah I got rid of that package too - I think this was an old commit.